update.js

```diff
@@ -0,0 +1,43 @@
+import { success, failure } from "./libs/response-lib";
+import * as dynamoDbLib from "./libs/dynamodb-lib";
+import User from './model/user';
+import HttpStatus from 'http-status-codes';
+import { userValidationCriteria, UUID_V1 } from './validators/user';
+import validate from 'validate.js';
+
+/**
+ * Update API Lambda for User model.
+ */
+export async function main(event, context) {
+  try {
+    if (!('pathParameters' in event)) {
+      return failure(HttpStatus.UNPROCESSABLE_ENTITY, {error : "ID Path Parameter required."});
+    }
+    const id = event.pathParameters.id;
+    const data = JSON.parse(event.body);
+    const errors = validate(data, userValidationCriteria);
+
+    if (!id) {
+      return failure(HttpStatus.UNPROCESSABLE_ENTITY, {error : "ID Path Parameter required."});
+    } else if (!id.match(UUID_V1)) {
+      throw {message : "Required UUID V1 format."};
+    } else if (errors) {
+      return failure(HttpStatus.BAD_REQUEST, {errors : errors});
```

Comment:
`{errors : errors}` can be shortened to just `{ errors }`

Reply:
Without the errors key its not as clear in the response that they are errors, this was my mentality.  But I think it could work without as well.